### ISSUE TITLE
Added a validator to remove "Run In Background" setting for HoloLens 2.

### ISSUE THREAD

**Issue by WikkidEdd** [microsoft/OpenXR-Unity-MixedReality-Samples#107]
The latest version of the package has a new validator which says "Run in Background is not recommended for HoloLens 2 applications"

Can you explain why this is not recommended?

**Comment by yl-msft** [comment]
When this "run in background" setting is enabled, unity engine will continue the rendering logic after the unity app is put in background on HL2.  Doing so might lead to unnecessary system cost for apps minimized. 

However, there was a few bugs in Unity earlier versions that confused the "focus" vs. "head tracking inputs", which leads to app stopped head tracking when app lost 3D focus, e.g. when user bring up start menu.   To prevent such bug, many earlier version of the HL2 unity apps turned on the "run in background" flag to workaround this problem.

In recent release of Unity 1.4.2 "Input System" package, there were fixes that allows head tracking continue to work when app lost 2D focus (i.e. keyboard focus), and 3D inputs like head/hand will continue regardless.  With this fix, we are recommending applications to remove the "run in background" setting and keep the app power footprint as small as possible on a mobile device like HoloLens 2.

**Comment by yl-msft** [comment]
@WikkidEdd  and others who is looking this or experiencing difficulties,  we recently discovered that Unity engine had some regression.  While we are working with Unity team on accessing the scope of regression, roughly there's some version after Unity 2020.3.27f1 to latest 40f1 introduced a bug that when "Run in background" is not enabled in the project settings, you might experience some problem on rendering.   Here's a few examples:

1. in unity editor play mode, whenever use click out of unity's window (put focus to another window), for example open a notepad, the XR rendering stops in both unity game view as well as XR view in HMD.

1. UWP XR app on desktop:   When running a UWP XR app, unless user us Win+Y to put keyboard focus inside XR, the UWP app won't render and HMD has black screen.

1. HL2 app with interactive carry:  When running a UWP XR app on HL2, when user start interactive carry, e.g. open Settings page as a slate overlay on top of the unity app, unity will stop rendering.


We are engaging Unity team to fix this issue, but in the meantime, we are making a change to our [1.6.0 MR openxr plugin](https://github.com/microsoft/OpenXR-Unity-MixedReality-Samples/releases/tag/v1.6.0) to revert the recommendation of this settings and help user's project to temporarily enable "Run in background" to avoid problems at the moment.   

When we confirmed fixes in a newer unity engine, we will modify our validation to help setup project settings properly.  Keep tuned
